src/app/kbv/controllers/single-amount-question.js

```diff
@@ -52,7 +53,12 @@ class SingleAmountQuestionController extends BaseController {
         return callback(err);
       }
 
-      const userInput = req.body[req.session.question.questionKey];
+      let userInput = req.body[req.session.question.questionKey];
+      userInput = userInput && fields.stripSpaces(userInput);
+
+      if (req.session.question.questionKey === "rti-p60-earnings-above-pt") {
```

Comment:
Probably for a separate PR but at some point we probably want to put the question keys in some sort of enum so there's only one place to update them? something like our 'constants' in core-front - https://github.com/govuk-one-login/ipv-core-front/blob/main/src/constants/contexts.js

Reply:
I will create a seperate PR for this after